tests/telemetry/test_telemetry.py

```diff
@@ -65,23 +33,24 @@ def test_telemetry_enabled_on_first_tracer_flush(test_agent_session, ddtrace_run
 span = tracer.trace("test-telemetry")
 span.finish()
     """
-    _, stderr, status, _ = ddtrace_run_python_code_in_subprocess(code, env=get_default_telemetry_env())
+    _, stderr, status, _ = ddtrace_run_python_code_in_subprocess(code)
     assert status == 0, stderr
     assert stderr == b""
     # Ensure telemetry events were sent to the agent (snapshot ensures one trace was generated)
     # Note event order is reversed e.g. event[0] is actually the last event
-    events = _assert_dependencies_sort_and_remove(
-        test_agent_session.get_events(), is_request=False, must_have_deps=False
-    )
+    events = test_agent_session.get_events()
 
-    assert len(events) == 4
-    assert events[0]["request_type"] == "app-closing"
-    assert events[1]["request_type"] == "app-integrations-change"
-    assert events[2]["request_type"] == "app-started"
-    assert events[3]["request_type"] == "generate-metrics"
+    assert len(events) == 5
+    # Generate metrics is sent after app-closed event. This is because the span aggregator is shutdown after the
```

Comment:
Should this be flipped to reflect the ordering you have in the assertions?
```suggestion
    # app-closed is sent after the generate-metrics event. This is because the span aggregator is shutdown after the
```